resources/tentative/charts/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```

Comment:
Can we call this one developer.html or something and the other one index.html?

Reply:
I can rename this one, but I'd prefer to keep the other one with the original name, so that we can add more html files for more workloads (like I did for chart.js).

Reply:
Also note that I followed the same structure than in `editors`.